docker-compose.yml

```diff
@@ -48,5 +45,22 @@ services:
       interval: 1s
       timeout: 6s
       retries: 20
+  kafka_consumer:
+    build: .
+    command:
+      - bash
+      - -c
+      - |
+        python manage.py kafka_consumer --debug
```

Comment:
So, just to make sure I get this, we are using the very same code AND database in both apps, but in one we run the server and in the other we run the kafka_consumer. Is that to better use the compute resources or to better manage the different components in case one of them fail?

Reply:
That's right. Ideally Docker Compose is built around the idea of only having one process running in each container. Whilst we could run this is the background under the `app` service we would indeed struggle with handling error states extra. I also think it's cleaner to have the consumer logs in a separate feed.

Ideally we might also run the ssh server in the `drunc` service as a separate service but there is a hard coded restriction in the dummy_boot command that the process must be started on localhost.